airbyte-integrations/connectors/source-mssql/README.md

```diff
@@ -0,0 +1,10 @@
+# How to use MsSQL Script?
```

Comment:
Nitpick. I'd suggest to give this doc a generic H1 title, because this is a README for the entire module. You can move what you wrote under a performance test H2 title. Something like:

```suggestion
# MsSQL (SQL Server) Source

## Performance Test

### Use MsSQL script to populate the benchmark database
```

Reply:
updated it